### ISSUE TITLE
Invalid characters in hasura namespace template

### ISSUE THREAD

**Issue by sbienkow-ninja** [platyplus/platyplus#9]
Trying to template hasura chart, results in:
```
Error: YAML parse error on hasura/templates/config-map-metadata.yaml: error converting YAML to JSON: yaml: line 4: did not find expected ',' or '}'
```
When opening the mentioned file, I can see:
![image](https://user-images.githubusercontent.com/85104051/128876882-02fb73f5-3fa8-40ae-8618-560aa8ae7d92.png)

Looks like somehow #8 introduced invalid characters there, causing the template to not render.
CC: @preiddygarcia

**Comment by ankit-ls** [comment]
@plmercereau @preiddygarcia I have submitted a PR #10 to fix this issue.

It would be great if you could merge and release a patch for it.

**Comment by onkarhoysalalivspace** [comment]
This is super useful, @ankit-ls . @plmercereau @preiddygarcia Request you to review and merge! This will solve a number of issues

**Comment by ankit-ls** [comment]
This issue can be closed now. It is fixed via #10 .

Please use the latest available chart versions.
